doc/server/tutorial.rst

```diff
@@ -216,6 +216,46 @@ assignments that do not change the value will still be reported). This default
 can be changed by passing `auto_strategy` and `auto_strategy_parameters` when
 constructing the sensor.
 
+Aggregate Sensors
+-----------------
+To provide a sensor which has its reading derived from that of a set of other
+sensors, such as a total, average or general "device status" sensor, the
+:class:`.AggregateSensor` class allows the user to implement this. The user
+should subclass :class:`.AggregateSensor`, and implement
+:func:`.~AggregateSensor.update_aggregate` in order to define the logic which will
+be used.
+
+For example:
+
+.. code:: python
+
+    class Total(AggregateSensor):
+        def __init__(self, target):
+            super().__init__(target=target, sensor_type=int, name="total")
+
+        def update_aggregate(self, updated_sensor, reading, old_reading):
+            if update_sensor is None:
+                # Instantiation, set a fresh value.
+                return Reading(time.time(), Sensor.Status.NOMINAL, 0)
+            if reading is None:
+                # The sensor is being removed from the set.
+                return Reading(updated_sensor.timestamp, Sensor.Status.Nominal, self.value - updated_sensor.value)
```

Comment:
`Nominal` should be `NOMINAL` - please check that the rest of the code actually functions. It might be an idea to add to the `examples/` directory, either adding this to the existing `example_server.py`, or creating a separate example.